src/docs/generate_oci_doc_yaml.py

```diff
@@ -262,6 +262,12 @@ def build_releases_data(
             release_data["tags"] = digest_tags.remove(channel_tag) or digest_tags
             # Get architecture list
             release_data["architectures"] = self.get_arches(channel_tag)
+            # Set the support date
+            if all_tracks.get(track_base):
+                eol = parser.parse(all_tracks[track_base])
+                release_data["support"] = {
+                    "until": eol.strftime("%m/%Y")
```

Comment:
Don't you need the day as well?

Reply:
we haven't been that granular for the Ubuntu image (https://gallery.ecr.aws/ubuntu/ubuntu)